src/colvarproxy.h

```diff
@@ -682,6 +697,9 @@ class colvarproxy
   /// Track which features have been acknowledged during the last run
   size_t features_hash;
 
+  /// Queue of config strings or files to be fed to the module
+  void *config_queue_;
```

Comment:
How would you about it without having to include the headers that define the involved templates, i.e. `list`, `utility` and `string` in `colvarproxy.h`?  I think you would need to declare a new class to hide those, which ends up being just as verbose.  If you feel very strongly about one preference vs. the other, please confirm.

Reply:
Alternatively, this can be moved to a class under the module's namespace, i.e. outside of the proxy where verbosity is less of a problem. But the module would then need to always be created immediately together with the proxy, meaning more refactoring. Can you live with this until then?

Reply:
I've just pushed a variant using the preprocessor in an arguably inelegant way. I find that slightly preferable to the void pointer version - maybe it's a matter of taste.